src/services/myPatients.ts

```diff
@@ -148,9 +148,7 @@ export const fetchMyPatients = async (): Promise<CohortData | undefined> => {
     const cohortData: CohortData = {
       name: 'Mes Patients'
     }
-    const patients = getApiResponseResources(
-      await api.get<FHIR_API_Response<IPatient>>(`/Patient?_count=700${API_RESOURCE_TAG}`)
-    )
+    const patients = getApiResponseResources(await api.get<FHIR_API_Response<IPatient>>(`/Patient?_count=700`))
```

Comment:
Maybe we should store the value of count somewhere in a global constant since we use it in a lot of requests